data/coalition/coalition missions.txt

```diff
@@ -3111,3 +3111,129 @@ mission "Coalition: Outskirts Gauger Ready"
+			`	"Greetings, <first> <last>. This other human you have brought, who she is, may I ask you?"` # heliarch
+			`	Ann clears her throat. "I don't know who you are, but my name is Ann Oryza Sativa."`
+			`	Once Ann finishes her introduction, you speak. "She is my friend. Back in my space, she hasn't been a successful author. I brought her here, hoping she would find success writing books for the Coalition.", you explain.`
+			`	"What kind of books does she write?", asks the Kimek.` # heliarch
+			`	"Oh, I write science fiction books," says Ann.`
+			`	The Kimek begin conversing amongst themselves in their native language for a good ten to twenty seconds before they move to face you again.`
+			`	"Visit Coalition space, we will allow Ann to. Alongside you, <first>, welcome here, she is, for as long as no trouble, you cause. But first, register she must."`
+			`	"Register?", asks Ann.`
+			`	"You have to register yourself with the Coalition before you can do most stuff here. I'll take you to the Ring of Friendship, that's where you can complete the registration process."`
+			`	"Okay.", says Ann. "I'll explore the spaceport a little bit to find some inspiration. Get your ship ready and take me to the Ring of Friendship once you're ready. I'll meet you in the spaceport."`
+
+mission "Sativa: Struggling Author 2"
+	name "Help Ann Register"
+	description "Take Ann Oryza Sativa to the <destination> so she can register with the Heliarchs."
+	source "Ki Patek Ka"
+	destination "Ring of Friendship"
+	to offer
+		has "Sativa: Struggling Author 1: done"
+	on offer
+		conversation
+			`	You walk through the spaceport, looking for Ann. She is at a restaurant, but has ordered nothing with a disgusted look on her face. "These aliens! They vomit on their food and slurp all the soup! Disgusting!", she mutters.`
+			choice
+				`	"That's just how they eat."`
+				`	"My ship still has a decent amount of food you can eat."`
+				`	"There should be another restaurant."`
+			`	"<first>! It's you again!", says Ann in an excited tone. "Is the ship ready? Remember, we need to go to the <destination> so I can get registered with the Heliarchs.`
+			choice
+				`	"It's all ready now. Come."`
+			`	Ann stands up from her chair and begins following you to the ship. She settles into her bunk as you prepare for a trip to the <destination>.`
+				accept
+	on enter "Quaru"
+		conversation
+			`	As you enter <system>, Ann walks out of her bunk and finds a spot on the side of the bridge, to peer into space. "What a massive ringworld!", she exclaims.`
+			choice
+				`	"It's massive, isn't it? They actually stole it from the Quarg, who they rebelled against roughly six thousand years ago."`
+				`	"I'm used to it."`
+			`	"Awesome! This is going to be a great setting and certainly a source of inspiration for my next novel!", says Ann, her voice still in an enthusiastic and excited tone.`
+			`	"This must be where I'm supposed to register, right?", asks Ann.`
```

Comment:
```suggestion
			`	"Awesome! This is going to be a great setting and certainly a source of inspiration for my next novel!", says Ann, her voice still in an enthusiastic and excited tone. "This must be where I'm supposed to register, right?" asks Ann.`
```
Can just join these two. If possible, try to avoid really short lines like her question here, if you can attach it to a line above or below it.

Reply:
And yes, I've joined that in a3ff8fd06dc73ecb5ca957f617fe856807a18ecd and also told Ann to prepare her identity documents.